src/features.rs

```diff
@@ -524,6 +524,51 @@ pub mod vsock {
     impl crate::FeatureBits for F {}
 }
 
+pub mod balloon {
+    use crate::le128;
+
+    feature_bits! {
+        /// Traditional Memory Balloon Device Feature Bits
+        #[doc(alias = "VIRTIO_BALLOON_F")]
+        pub struct F: le128 {
+            /// Host has to be told before pages from the balloon are used.
+            #[doc(alias = "VIRTIO_BALLOON_F_MUST_TELL_HOST")]
+            const MUST_TELL_HOST = 1 << 0;
+
+            /// A virtqueue for reporting guest memory statistics is present.
+            #[doc(alias = "VIRTIO_BALLOON_F_STATS_VQ")]
+            const STATS_VQ = 1 << 1;
+
+            /// Deflate of the balloon is always? permitted on guest out of memory condition.
+            ///
+            /// TODO: Spec is a bit confusing on this feature, see <https://github.com/oasis-tcs/virtio-spec/issues/228>
```

Comment:
Let's stick to the spec first and add notes afterwards:

```suggestion
            /// Deflate balloon on guest out of memory condition.
            ///
            /// <div class="warning">
            ///
            /// The specification is a bit confusing on this feature, see <https://github.com/oasis-tcs/virtio-spec/issues/228>.
            ///
            /// </div>
```